terraform/main.tf

```diff
@@ -0,0 +1,79 @@
+
+module "network" {
+  source = "./modules/network"
+  vpc_cidr      = var.vpc_cidr
+  subnet_1_cidr = var.subnet_1_cidr
+  subnet_2_cidr = var.subnet_2_cidr
+}
```

Comment:
Question: Do we need to add a NAT gateway 🧐

Reply:
😭 i’m scared to touch the code again

Reply:
Also I only have public subnets so a NAT gateway wouldn’t  add additional security if that’s what you were thinking.

definitely if i had a private subnet

Reply:
Okay nice, yh i was just wondering